docs/articles/documentation/test-api/assertions/index.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: docs
+title: Assertions
+permalink: /documentation/test-api/assertions/
+checked: false
+---
+# Assertions
+
+To check if the state of the tested webpage matches the one you expect to see, use *assertions*.
+
+TestCafe provides a comprehensive set of assertions and all of them based on Behavior Driven Development style (BDD-style).
+See [Assertion API](assertion-api.md).
+
+This topic contains the following sections.
+
+* [Using Assertions](#using-assertions)
+* [Assertion timeout](#assertion-timeout)
+* [Assertion options](#assertion-options)
+  * [options.timeout](#optionstimeout)
+
+## Using Assertions
+
+The following example demonstrates how to check whether two values are equal or not.
```

Comment:
I guess it shouldn't be an `example` here we demonstrate the general structure of assertions. Instead of `eql` we should use `[assertionType]` placeholder and link to the possible assertions types.